### PR TITLE
feat(Documentation): quick setup with virtual environment

### PR DESCRIPTION
**Pull request by DoodahProductions**

### CODE REVIEW

File: QuickSetup.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Mac setup
+
+### Cloning the project
+First, you'll need to clone the project on your machine.  
+
+Pommerman project is hosted on GitHub.  
+GitHub is a web-based hosting service for version control using git. You can create a free  
+account on GitHub, and clone the project to your local machine.
+
+GitHub url : https://github.com/MultiAgentLearning/playground
```
Comment (cinjon) [inline QuickSetup.md]: They're already at this repository, so this section is all redundant.

File: QuickSetup.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Setup virtual environment
+You'll need a virtual python environment to run the project.  
+
+You can follow this guide to do so http://sourabhbajaj.com/mac-setup/Python/virtualenv.html
+
+Note that you probably want to install virtual environment as root user using sudo command
+```$ sudo -H pip3 install virtualenv```
```
Comment (cinjon) [inline QuickSetup.md]: I don't think Sudo is the suggested way of using virtualenv. 
Please point them to a proper install page like https://virtualenv.pypa.io/en/stable/

Comment (DoodahProductions) [reply QuickSetup.md]: Seems sudo is suggested following your link : https://virtualenv.pypa.io/en/stable/installation/

File: QuickSetup.md (hunk 1 above)
Comment (DoodahProductions) [reply QuickSetup.md]: Done

File: QuickSetup.md (hunk 2 above)
Comment (cinjon) [reply QuickSetup.md]: Are you referring to the brackets around the sudo? I don't think that that is saying to use sudo; it's saying you can.

This SO answer - https://askubuntu.com/questions/234758/how-to-use-a-python-virtualenv-with-sudo - explains a little bit about why it's not preferred.

Comment (DoodahProductions) [reply QuickSetup.md]: Done

**Comment by cinjon** [comment]
Do you mind incorporating this into the GettingStarted doc --> https://github.com/MultiAgentLearning/playground/blob/master/GettingStarted.md please?

**Pull request closed without merging**